src/pages/docs/projects/variables/aws-account-variables.md

```diff
@@ -42,9 +53,24 @@ Each of the above properties can be referenced in PowerShell.
 Write-Host 'AwsAccount.Id=' $OctopusParameters["aws account"]
 Write-Host 'AwsAccount.AccessKey=' $OctopusParameters["aws account.AccessKey"]
 
+# For an OpenId Connect account
+Write-Host `AwsAccount.RoleArn=` $OctopusParameters["aws account.RoleArn"]
+Write-Host `AwsAccount.SessionDuration=` $OctopusParameters["aws account.SessionDuration"]
```

Comment:
Do we want to add the JWT example here?

Reply:
the jwt is sensitive, so it won't write anything useful anyway. I noticed that the other example also leaves out the secret too